command/start.go

```diff
@@ -281,24 +274,6 @@ func (c *startCommand) Run(args []string) int {
 			return
 		}
 
-		switch ctrl := ctrl.(type) {
```

Comment:
This block of code has definitely confused me in the past, so kind of nice to see it removed and the logic simplified! 😄